packages/block-editor/src/components/global-styles/use-global-styles-output.js

```diff
@@ -900,6 +916,16 @@ export const toStyles = (
 	return ruleset;
 };
 
+export function customSvgFilters( filters ) {
+	return filters.flatMap( ( filter ) => (
+		<DuotoneFilter
```

Comment:
Looks like the SVG is showing up after the preset ones, but I don't see it on my images. They also show the filter on their css. I have this on my theme.json:


```
"core/image": {
				"filter": {
					"duotone": [
						"#FF0000",
						"#000000"
					]
				}
			},
```

@ajlende do you have any clue what I might be missing?